Assignment_4/AccountOverdrawDemo.java

```diff
@@ -0,0 +1,26 @@
+public class AccountOverdrawDemo implements Runnable {
+    private Account acct = new Account();
+
+    @Override
+    public void run() {
+        for (int x = 0; x < 5; x++) {
+            makeWithdrawal(60);
+            if (acct.getBalance() < 0) {
+                System.out.println("account is overdrawn!");
+            }
+        }
+    }
+    private void makeWithdrawal(int amt) {
+        if (acct.getBalance() >= amt) {
+            System.out.println(Thread.currentThread().getName() + " is going to withdraw "+amt+", current balance : "+acct.getBalance());
+            try {
+                Thread.sleep(100);
+            } catch (InterruptedException ex) {
```

Comment:
In this catch block you can print the exception so if any error throw in this exception then you get the log